subprojects/core-api/src/main/java/org/gradle/api/artifacts/ComponentMetadata.java

```diff
@@ -57,4 +57,13 @@
      */
     List<String> getStatusScheme();
 
+    /**
+     * Returns the dependencies of the component. These can be used
+     * to inspect them for making adjustment decisions in component
+     * metadata rules.
+     *
+     * @return the dependencies of the component
+     * @since 4.4
+     */
+    List<ModuleVersionIdentifier> getDependencies();
```

Comment:
Why would we only give access to the identifiers? What if I need to look whether the dependency is strict or whether it has some excludes etc?